src/api/public/apidocs-new/paths/build_project_repository_arch_package_log.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  description: This endpoint returns the last log file for a specific build artifact
+  operationId: getBuildProjectRepositoryArchPackageLog
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+  responses:
+    '200':
+      description: |
+        This endpoint returns the log file contents
```

Comment:
```suggestion
        This endpoint returns the log file content
```

Reply:
I think its `file contents` ([like here](https://stackoverflow.com/q/29881063))

Reply:
Well, I'm not sure. I'll accept your suggestion